pretty.c

```diff
@@ -393,20 +388,20 @@ static void print_fb(struct json_object *obj, const char *prefix)
 
 	if (has_legacy) {
 		printf("%s" L_VAL "Pitch: %"PRIu32"\n", prefix,
-			(uint32_t)get_object_uint64(pitch_obj));
+			(uint32_t)json_object_get_uint64(pitch_obj));
```

Comment:
Is there `json_object_get_uint32`? If so, we could just use that.

Reply:
There is no json_object_get_uint32.